Clarify Ambiguities between OpenID Federation and RFC 5280 (#67)
peppelinux/draft-demarco-acme-openid-federation: draft-demarco-acme-openid-federation.md

This document defines how X.509 certificates associating a given OpenID
Federation Entity with a key included in that Entity's Configuration can be

> issued by a X.509 Certification Authority through the ACME protocol to the

Reviewer: ```suggestion
issued by an X.509 Certification Authority through the ACME protocol to the
```

  `/.well-known/acme-challenge/{token}` endpoint.

- It defines how the OpenID Federation Subordinate Statements can be used for the

>   publication of the X.509 Certificates, by a Certification Authority, that

Reviewer 2: I think this might be an incorrect change, as the existing text is talking about how to reflect the X.509 certificates back into OpenID Federation subordinate statements. We could fix this by instead by using the term "Superior Entity", defined in OpenID Federation.

Author: I don't think it's incorrect, by my understanding, as it would be the CA doing the work to publish after having issued the certificate, but I agree Superior Entity is much better.
```suggestion
  publication of the X.509 Certificates, by a Superior Entity, that
```

# Terminology

The terms "Federation Entity", "Trust Anchor", "Entity Configuration",

> "Subordinate Statement", "Trust Mark" and "Trust Chain" used in this

Reviewer 2: We should add "Superior Entity" to this list.

Author: In https://github.com/peppelinux/draft-demarco-acme-openid-federation/pull/67/commits/66aba9b371a333bc1e993f4ac59290690aabc1de

# Publication of the Certificates within the Federation

> **TBD**, when the Certificate Issuer is the Superior Entity, the X.509

Reviewer 3: ```suggestion
When the Certificate Issuer is the Superior Entity, the X.509
```
